sources/packages/api/src/route-controllers/institution-locations/models/institution-location.dto.ts

```diff
@@ -64,3 +65,17 @@ export interface InstitutionLocationsSummaryDto {
     postalCode: string;
   };
 }
+
+/**
+ * Possible status for a designation location agreement.
+ */
+export enum DesignationLocationAgreementStatus {
```

Comment:
Since this enum is used for location and institution status, can we call it just `DesignationStatus`?
Unfortunately, the DB enum was wrongly named `DesignationAgreementStatus` when it was supposed to be `DesignationAgreementApprovalStatus`. I believe that this one was the one to be named as `DesignationAgreementStatus` 😉